opt/test/test_cell_color_by_features.py

```diff
@@ -0,0 +1,16 @@
+from scopeserver.dataserver.utils.cell_color_by_features import CellColorByFeatures
+import numpy as np
+
+
```

Comment:
This is a great place for some property based testing. I suggest a test something like this (note you'll need to `poetry add --dev hypothesis hypothesis[numpy]`

```python
from hypothesis import given
from hypothesis.strategies import integers
from hypothesis.extra.numpy import arrays

from scopeserver.dataserver.utils.constant import LOWER_LIMIT_RGB, UPPER_LIMIT_RGB

@given(arrays(np.intc, shape=integers(0, 50), elements=integers(-1000, 1000)),
       integers(0, 1000),
       integers(0, 1000))
def test_normalise_vals2(vals, v_min, v_max):
    normalised = CellColorByFeatures.normalise_vals(vals, v_max, v_min)
    assert np.amin(normalised) >= LOWER_LIMIT_RGB
    assert np.amax(normalised) <= UPPER_LIMIT_RGB
```

Reply:
This looks amazing